tests/e2e/driver/ChromeDriver.ts

```diff
@@ -21,6 +21,7 @@ export class ChromeDriver implements IDriver {
 
 	constructor() {
 		const options: Options = this.getDriverOptions();
+		options.setLoggingPrefs({performance: 'ALL'})
```

Comment:
looks like prettier script was not run before commit

Reply:
It should be fixed in my latest commit